src/schematics/ng-add/index.ts

```diff
@@ -148,5 +148,7 @@ function getPugLoaderRules(): string {
     {
       test: ${partialRegex},
       loader: 'pug-loader'
-    },`.replace(/\s+/gm, ' ');
+    },`
+      .replace(/\s+/gm, ' ')
+      .replace(/^\s/, '');
```

Comment:
This change is not necessary.
Please focus only on the current problem.

Reply:
I agree, but it looks better without first space (imho).